### DISCUSSION TITLE
How to move from HACS to Core intergation

### DISCUSSION THREAD

**Discussion by Eisoniq** [humbertogontijo/homeassistant-roborock#658] [Q&A]
When humbertogontijo created this integration in HACS I quickly started using it with automatons and it works great, however it seems to Core version is getting more development now so I want to move to that version.
Is there a recommended way to move from a HACS integration to the Core one without losing anything and having to setup my automatons again?

**Comment by Trilis29** [comment]
You should not loose anything. Do backip, remove jacs integration make full system reboot (hard reboot) and install core integration.

**Reply by Rolfgithub2** [reply]
I don't understand.
I want to by a Roborock Q7 L5,
You say I have to use the core installation, but that does not support much, Everything new is not supported, from Q7 on there is a change in protocol is written on the information page of this core integration... 
This core integration is obviously not a alternative it is not maintained ....
I don't understand That here I am advised not to install this custom version, because of the lack of maintenance.

When I read this and search the web, I can't determine if this roborock vacuum is able to connect to HA.
Please can you help to clarify if roborock is worth buying, as I now understand all new devices from Roborock are not connectable to HA..

**Comment by popy2k14** [comment]
Tried this and ended up with complete messed up dashboard. 

Must do this when I have time and migrate every single dashboard, automation, ...

**Comment by r4dian** [comment]
The core version does not support q7 robot, does this version support it?

(q7 m5, specifically)
